netjsonconfig/backends/raspbian/templates/hostapd.jinja2

```diff
@@ -0,0 +1,26 @@
+{% for wireless in data.wireless %}
+    # config: /etc/hostapd/hostapd.conf
+
+    interface={{ wireless.get('ifname') }}
```

Comment:
why not `{{ wireless.ifname }}` instead? Jinja2 should allow you this syntax which is more concise and easy to read